zingo-sync/src/client.rs

```diff
@@ -0,0 +1,49 @@
+//! Module for handling all connections to the server
+
+use std::ops::Range;
+
+use zcash_client_backend::proto::{compact_formats::CompactBlock, service::BlockId};
```

Comment:
It's important to double check with zingolib to see if there's another definition of `CompactBlock` dangling around.

Reply:
yeah good idea. i have checked and this is the same one used in zingolib currently and in LRZ etc. as defined in the proto files